### DISCUSSION TITLE
Beta8 -> ReflectionTypeLoadException

### DISCUSSION THREAD

**Discussion by SaschaHerrmann** [Dotnet-Boxed/Templates#35] [Q&A]
I like the boilerplate very much, was working fine with beta7.

After an update to beta8 with the change to kestrel and boilerplate version 1.2.6 the page will not start.
I stripped out everything using Boilerplate.Web.Mvc and related and the project would still not work.

The only way to get the project to load again is to remove the boilerplate entirely from project.json:
"Boilerplate.Web.Mvc6": "1.2.6"

**Comment by RehanSaeed** [comment]
Boilerplate has not been upgraded to Beta 8 yet. I will get it out some time this week. There will be a big Beta 7 update first though.

**Comment by SaschaHerrmann** [comment]
Looking forward to it. Tank you :)

**Comment by RehanSaeed** [comment]
Great!
